### PR TITLE
EVG-14261 more instrumentation

### PR DESCRIPTION
**Pull request by john-m-liu**
the actual route takes on the scale of 50 ms, so the performance problem has to be in the middleware

### CODE REVIEW

File: service/middleware.go [hunk 1]
```diff
@@ -393,6 +423,22 @@ func (uis *UIServer) LoadProjectContext(rw http.ResponseWriter, r *http.Request)
 		})
 	}
 
+	grip.Debug(message.Fields{
+		"ticket":      "EVG-14261",
+		"id":          id,
+		"step":        "LoadContext",
+		"func":        "LoadProjectContext",
+		"duration_ns": time.Since(start).Nanoseconds(),
```
Comment (ablack12) [inline service/middleware.go:431]: The other PR had duration as well as duration_ns, if that was useful and you want to have both

**Review by ablack12 — COMMENTED** [review]
Is it going to cause future issues if we modify our copy of the vendored library?

File: service/middleware.go (hunk 1 above)
Comment (john-m-liu) [reply service/middleware.go:431]: nope, they're both the same so I kept 1

**Review by john-m-liu — COMMENTED** [review]
the only issue it would cause is revendoring would delete the changes made here. I'm planning on reverting the changes once we know what's going on anyway so that's fine

**Review by ablack12 — APPROVED** [review]

**Comment by john-m-liu** [comment]
evergreen merge

**Pull request merged**